build-data-capturing-maven-samples/capture-extension-version/maven-capture-extension-version.groovy

```diff
@@ -0,0 +1,11 @@
+import java.nio.charset.Charset
+import java.util.concurrent.TimeUnit
+import com.gradle.maven.extension.api.scan.BuildScanApi
+
+/**
+ * This Groovy script captures the OS processes as reported by the OS 'ps' command,
+ * and adds these as a custom value.
+ */
+URL url = BuildScanApi.class.getClassLoader().getResource("com.gradle.scan.plugin.internal.meta.buildAgentVersion.txt");
+String buildAgentVersion = Resources.toString(url, Charsets.UTF_8);
```

Comment:
Is it indeed UTF_8 or just ASCII?

Reply:
You are correct that it is ASCII but I have switched to a different api `Scanner` since `Resources` is part of guava and not available here.

Reply:
Got it.